cfgov/v1/tests/atomic_elements/organisms/test_organisms.py

```diff
@@ -216,3 +216,18 @@ def test_item_introduction(self):
         response = django_client.get('/learn/')
         self.assertContains(response, 'Item Introduction')
         self.assertContains(response, 'Item introduction body')
+
+    def test_html_block(self):
+        """ HTML Block correctly renders HTML on a Browse Page"""
+        browse_page = BrowsePage(
+            title='Browse Page',
+            slug='browse',
+        )
+        browse_page.content = StreamValue(
+            browse_page.content.stream_block,
+            [atomic.html_block],
+            True
+        )
+        publish_page(child=browse_page)
+        response = self.client.get('/browse/')
```

Comment:
Minor thing, but this test will actually make a request to the URL in the `_atomic_helpers.html_block` value. This will fail if you run the tests without an internet connection (and I think we've also seen cases where Travis seems to block some outgoing requests). I'm not really sure of a nice and easy way to avoid this, short of mocking the requests call before running the test.